azure/azure-pr-teardown-pipeline.yml

```diff
@@ -0,0 +1,55 @@
+name: "$(SourceBranchName)+$(BuildID)"
+
+trigger: none
+pr: none
+
+resources:
+  repositories:
+    - repository: common
+      type: github
+      name: NHSDigital/api-management-utils
+      ref: refs/heads/edge
+      endpoint: NHSDigital
+
+variables:
+  - template: project.yml
+
+jobs:
+  - job: teardown
+    displayName: PR Teardown
+    timeoutInMinutes: 30
+    pool:
+      name: 'AWS-ECS'
+      vmImage: 'ubuntu-latest'
+    steps:
+      - checkout: self
+
+      - bash: |
+          echo $(action_pr_number)
+          WORKSPACE="pr-"$(ACTION_PR_NUMBER)
+          echo $WORKSPACE
+          echo "##vso[task.setvariable variable=WORKSPACE]$WORKSPACE"
+          echo "##vso[task.setvariable variable=ACTION_PR_NUMBER]$(ACTION_PR_NUMBER)"
+        displayName: Set workspace tag
+
+      - template: "azure/components/aws-assume-role.yml@common"
+        parameters:
+            role: "auto-ops"
+            profile: "apim-dev"
+            aws_account: "dev"
+
+      - bash: |
```

Comment:
Is this necessary? Can't find any usage of it. I think it can be export env var like any other.

Reply:
No in teardown pipeline, we can't access variables like in other pipelines